vignettes/hys_law.Rmd

```diff
@@ -0,0 +1,191 @@
+
+Additional modifications can be made such as:
+
+- Parameter to indicate worsening of condition
+- Any sort of baseline/post-baseline based analysis
+- Flags for other lab values like ALP if modified in above too
+
+```{r}
+hylaw_records_pts_visits <- hylaw_records %>%
+  select(STUDYID, USUBJID, TRT01A) %>% # add AVISIT, ADT for by visit
+  distinct()
+
+hylaw_records_fls <- hylaw_records %>%
+  select(STUDYID, USUBJID, TRT01A, CRIT1FL, BILI_CRITFL) %>% # add AVISIT, ADT for by visit
+  distinct()
+
+hylaw_params <- derive_param_exist_flag(
+  dataset_adsl = hylaw_records_pts_visits,
+  dataset_add = hylaw_records_fls,
+  condition = CRIT1FL == "Y" & BILI_CRITFL == "Y",
+  false_value = "N",
+  missing_value = "N",
+  subject_keys = exprs(STUDYID, USUBJID, TRT01A), # add AVISIT, ADT for by visit
+  set_values_to = exprs(
+    PARAMCD = "HYSLAW",
+    PARAM = "ALT/AST >= 3xULN and BILI >= 2xULN"
+  )
+)
+```
+
+```{r, echo = FALSE}
+hylaw_params %>%
+  arrange(desc(AVAL)) %>%
+  dataset_vignette()
+```
+
+
+The last step would be binding these rows back to whatever previous dataset is appropriate based on your data specifications, in this case, it would be best suited to bind back to our `adlb_annotated` object.
+
+## Conclusion {#conclusion}
+
+```{r}
+adlbhy <- adlb_annotated %>%
+  bind_rows(hylaw_params)
+```
+
+```{r, echo = FALSE}
+dataset_vignette(adlbhy)
+```
+
+Here we demonstrated what is the base-case that may be asked of as a trial programmer. The reality is that Hy's Law and assessing potential DILI events can get rather complex quite quickly. Differences in assessment across organizations and specific trials might require modifications, which may include:
+
+ - additional CRITX and CRITXFL columns for different cutoffs like 5xULN, 10xULN, 20xULN
+ - checking for elevated values of additional labs like alkaline phosphatase (ALP) 
+ - appearance of certain adverse events associated with some of these elevated lab-values
+ - different criteria cutoffs that depend on baseline values or characteristics
+ - other parameters such as worsening of condition
+
+We hope by demonstrating the flexibility of `admiral` functions and using a general workflow to create the necessary parameters for an `adlbhy`, that creating this final dataset becomes simplified and easily scalable. Ideally, this is ready for your organization's standard macros or previous code for TLFs and outputs as well. This is our first attempt at breaking down and summarizing this topic. We welcome feedback and ideas to improve this guide!
```

Comment:
Most of our guides have example scripts at the bottom.  can we list out the one you created?  
![image](https://user-images.githubusercontent.com/10111024/219484110-1dd9bfc4-0ca7-4adc-942a-38bb41bc0478.png)